shared/libraries/opcuatms/opcuatms_client/src/objects/tms_client_component_impl.cpp

```diff
@@ -12,20 +12,33 @@ using namespace daq::opcua;
 
 template <class Impl>
 ErrCode TmsClientComponentBaseImpl<Impl>::getActive(Bool* active)
-{
-    return daqTry([&]() {
+{   
+    try
+    {
         *active = this->template readValue<IBoolean>("Active");
-        return OPENDAQ_SUCCESS;
-    });
+    }
+    catch(...)
+    {
+        *active = true;
+        auto loggerComponent = getLoggerComponent();
+        LOG_W("Failed to get active of component \"{}\"", this->globalId);
+    }
+    return OPENDAQ_SUCCESS;
 }
 
 template <class Impl>
 ErrCode TmsClientComponentBaseImpl<Impl>::setActive(Bool active)
 {
-    return daqTry([&]() {
+    try
+    {
         this->template writeValue<IBoolean>("Active", active);
-        return OPENDAQ_SUCCESS;
-    });
+    }
+    catch(...)
+    {
+        auto loggerComponent = getLoggerComponent();
+        LOG_W("Failed to set active of component \"{}\"", this->globalId);
+    }
+    return OPENDAQ_SUCCESS;
```

Comment:
I think setters should return OPENDAQ_IGNORED, and log the fact at debug or info level. The core implementation does it in a similar manner (at info). The previous comment about returning OPENDAQ_IGNORED was aimed at getters, not setters.

```
...
        if (lockedAttributes.count("Name"))
        {
            if (context.assigned() && context.getLogger().assigned())
            {
                const auto loggerComponent = context.getLogger().getOrAddComponent("Component");
                StringPtr descObj;
                this->getName(&descObj);
                LOG_I("Name of {} is locked", descObj);
            }

            return OPENDAQ_IGNORED;
        }

        this->name = name;
...
``` 

Getters could also be left at "warning level", but I have a feeling we might flood stdout with that. The log could be changed to:

`LOG_I("Failed to get active of component \"{}\". The default value was returned (\"true\")", this->globalId);`